Games/types/Mafia/effects/Quiet.js

```diff
@@ -0,0 +1,32 @@
+const Effect = require("../Effect");
+
+module.exports = class Quiet extends Effect {
+  constructor(lifespan) {
+    super("Quiet");
+    this.lifespan = lifespan || Infinity;
+  }
+
+  speak(message) {
+    message.recipients = [this.player];
+    message.modified = true;
+    message.parseForReview = this.parseForReview;
+  }
+
+  speakQuote(quote) {
+    quote.recipients = [this.player];
+    quote.modified = true;
+    quote.parseForReview = this.parseForReview;
+  }
+
+  speakWhisper(message) {
```

Comment:
this means he can only whisper to himself

Reply:
added a check for hearing whispers

im a not sure if I should actually put the ability to hear whispers above the other checks for silencing messages, thoughts?

Reply:
i don't get what this means

im a not sure if I should actually put the ability to hear whispers above the other checks for silencing messages, thoughts?

what are you concerned about, and what's the intended effect

Reply:
actually I think what I was asking is irrelevant, it won't make a difference

was just making sure that players with the quiet effect can still whisper even if they can't send normal messages